### PR TITLE
WAL-341 Add ECS container definitions

### PR DESCRIPTION
**Pull request by morgsmccauley**
This PR defines ECS container definitions for both contract-helper and indexer services. This is a step towards packaging task definitions along with application code within our CI pipeline.

### CODE REVIEW

File: ecs/indexer.container-definition.json [hunk 1]
```diff
@@ -0,0 +1,143 @@
...
+      {
+        "name": "BLOCK_TEMP_EMAIL_API_KEY",
+        "valueFrom": "arn:aws:ssm:{{AwsRegion}}:{{AwsAccountId}}:parameter/env/{{Environment}}/BLOCK_TEMP_EMAIL_API_KEY"
+      },
+      {
+        "name": "FUNDED_ACCOUNT_CREATOR_KEY",
+        "valueFrom": "arn:aws:ssm:{{AwsRegion}}:{{AwsAccountId}}:parameter/env/{{Environment}}/FUNDED_ACCOUNT_CREATOR_KEY"
+      },
+      {
+        "name": "INDEXER_DB_CONNECTION",
```
Comment (andy-haynes) [inline ecs/indexer.container-definition.json:90]: I think this might be the only variable we need for `indexer`

**Review by andy-haynes — CHANGES_REQUESTED** [review]
Nice job breaking these up, just a handful of hardcoded vars and some unneeded values in `indexer` (thank you for splitting those out! ❤️)

File: ecs/contract-helper.container-definition.json [hunk 2]
```diff
@@ -0,0 +1,147 @@
...
+        "name": "FUNDED_ACCOUNT_BALANCE",
+        "value": "350000000000000000000000"
+      },
+      {
+        "name": "DEBUG",
+        "value": "koa-ratelimit"
+      },
+      {
+        "name": "TOKEN_CONTRACTS",
+        "value": "meta.pool.testnet"
```
[the remark this answers is not in the document]
Comment (morgsmccauley) [reply ecs/contract-helper.container-definition.json]: This actually isn't used so I removed it. I did interpolate `NODE_URL` and `WALLET_URL` though :)

File: ecs/indexer.container-definition.json (hunk 1 above)
Comment (morgsmccauley) [reply ecs/indexer.container-definition.json:39]: Just this and a couple more. I've removed all which are unused within `indexer` :)

**Review by andy-haynes — APPROVED** [review]
🚀

**Comment by andy-haynes** [comment]
NB we're holding off merging this until we've reconsolidated the Jenkinsfiles so that we can iterate on deployment changes in PRs in a sane way.

**Pull request merged**